cli/src/lib.rs

```diff
@@ -72,6 +72,48 @@ pub struct Iroha {
     pub block_sync: AlwaysAddr<BlockSynchronizer>,
     /// Torii web server
     pub torii: Option<Torii>,
+    sumeragi_relay: AlwaysAddr<FromNetworkBaseRelay>,
+}
+
+use iroha_p2p::network::NetworkBaseRelayOnlinePeers;
```

Comment:
Rust supports imports inside functions. The reason being that you want to keep your imports as close to the place they are used as possible, if they are only used by that one piece of code.

There has to be a relay, otherwise there's a cyclical dependency between p2p and sumeragi. My instinct was to merge the two modules but since that would have been very destructive I chose this method instead.